src/stack_aggregator.py

```diff
@@ -506,8 +515,19 @@ def get_dependency_data(epv_set):
         if result:
             epv_list['result']['data'] += result['result']['data']
 
+    # # Identification of unknown dependencies
+    epv_data = epv_list['result']['data']
+    for k, v in dep_list.items():
+        known_flag = False
```

Comment:
I think you could use `for-else` construct here, instead of relying on this flag. But your way could be more readable for some people, so I don't mind keeping it this way.

Reply:
ok. thanks. I will keep it as it is